app/util/Youtube.scala

```diff
@@ -20,6 +20,7 @@ class YouTubeConfig @Inject()(config: Configuration) {
   lazy val clientSecret = config.getString("youtube.clientSecret").getOrElse("")
   lazy val refreshToken = config.getString("youtube.refreshToken").getOrElse("")
   lazy val contentOwner = config.getString("youtube.contentOwner").getOrElse("")
+  lazy val allowedChannels = config.getStringList("youtube.allowedChannels")
```

Comment:
nb: we'd need to update the conf files in S3 before merging if we don't provide a default via `getOrElse`